_posts/2015-05-06-[POSTMORTEM]-production-dowtime-deploys-nginx-proxy.markdown

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title:  POST-MORTEM Production services downtime after a deployement
+author: Charles Martinot
+date:   2015-05-06 16:52:00
+---
+From now on, we will publish our post-mortems here. We hope that what we learn 
+from fixing downtimes and incidents will help others in turn.
+
+If you're not doing post-mortems yet, you should. Here is a [good article][1] 
+explaining why and how to do them.
+
+### Description of the issue : 
```

Comment:
True, that's actually a French thing. It does look better with a space though so we all know what that means about French language.